aeon/dj_pipeline/spike_sorting.py

```diff
@@ -296,11 +312,20 @@ def make_compute(self, key, recording_file, output_dir, params, sorting_method):
         si_recording: si.BaseRecording = si.load(
             recording_file, base_folder=output_dir
         )
+        # load the pre-generated binary file - recording.dat
+        binary_rec = load_and_verify_binary_file(
+            binary_file_path=Path(recording_file).parent / "recording.dat",
+            se_recording_obj=si_recording)
+
         sorting_params = params["SI_SORTING_PARAMS"]
 
+        # explicitly set `skip_kilosort_preprocessing` to False
+        # to avoid SpikeInterface rerunning the `write_binary_recording` step
+        sorting_params["skip_kilosort_preprocessing"] = False
```

Comment:
Shouldn't this be `True` if we want to skip the preprocessing? [Default](https://github.com/SpikeInterface/spikeinterface/blob/171bc0cc5d9852e998464b03431b60ff3cbf50eb/src/spikeinterface/sorters/external/kilosort2_5.py#L62) is `False` 

```suggestion
        sorting_params["skip_kilosort_preprocessing"] = True
```

Reply:
Should we remove the following line in ephys_mock_ingestion.py as well?
https://github.com/SainsburyWellcomeCentre/aeon_mecha/blob/66dea030c45658272f5c035e9e6a3bf1e090bfc0/aeon/dj_pipeline/scripts/ephys_mock_ingestion.py#L352

Reply:
It actually needs to be `False` - see [here](https://github.com/SpikeInterface/spikeinterface/blob/813d9640eed6e4e28a411e37efaef67026b790e3/src/spikeinterface/sorters/external/kilosortbase.py#L131)
(it's a bit convoluted)